starter-code/src/functions-and-arrays.js

```diff
@@ -1,4 +1,15 @@
 // Find the maximum
+function maxOfTwoNumbers (num1, num2) {
+  var result = '';
+  if (num1>num2) {
+    result = num1;
+  } else if (num1<num2 ){
```

Comment:
Is this moment, mean that `num2` is greater or equal than `num1` so we can just put `result = num2`